src/main/java/se/liu/ida/hefquin/engine/data/mappings/impl/VocabularyMappingWrappingImpl.java

```
@@ -72,4 +72,10 @@ public Set<SolutionMapping> translateSolutionMappingFromGlobal( final SolutionMa
 		}
 		return result;
 	}
+
+	@Override
+	public boolean isEquivalenceOnly() {
+		return false;
```

No. Instead of simply returning false, throw an UnsupportedOperationException.